_posts/2018-10-11-utah-sgid-statewide-roads-data-layer-updates-10112018.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 10/11/2018
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2018-10-11
+categories:
+  - Featured
+tags:
+  - roads
+---
+
+AGRC recently completed updates to the SGID10.Transportation.Roads feature class that resides on the [Utah SGID ArcSDE database server]({{ "/sgid-database/" | prepend: site.baseurl }}). Updated roads data is also available as shapefiles and geodatabase files for download on the [SGID Roads and Highway System download page]({{ "/data/transportation/roads-system/" | prepend: site.baseurl }}). These updates are also reflected in [AGRC's address locators]({{ "/data/address-geocoders-locators/" | prepend: site.baseurl }}) and the [Utah Next-Generation 911 GIS Database]({{ "/data/911/" | prepend: site.baseurl }}).
+
+
+The following are highlights from this month's update.  
+
+Note: This update includes two separate road updates, spanning September and October:
+
+#### County Updates
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Cache County:** Obtained roads data on 08/13/2018. Previous update was on 06/27/2018.
+
+- **Davis County:** Obtained roads data on 09/10/2018. Previous update was on 08/14/2018.
+
+- **Salt Lake County (2 updates):** Obtained roads data on 08/09/2018 and 09/10/2018. Previous update was on 08/14/2018.
+
+- **San Juan County:** Obtained roads data on 08/16/2018. Previous update was on 01/2016.
+
+- **Summit Juan County:** Obtained roads data on 08/30/2018. Previous update was on 05/30/2018.
+
+- **Tooele Juan County:** Obtained roads data on 08/10/2018. Previous update was on 06/27/2018.
+
+- **Utah County (2 updates):** Obtained roads data on 08/09/2018 and 09/10/2018. Previous update was on 08/14/2018.
```

Comment:
Same with these...

Reply:
Again, odd dates.. but correct.  I double checked them.  Thanks again for the level of detail that you're checking this!  I appreciate it.